notebooks/ingestion/Loading.ipynb

```diff
@@ -0,0 +1,275 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading the data into the DB.\n",
```

Comment:
I think we need to split these. 
LoadEntities.ipynb
LoadImages.ipynb 
etc....
When we combine these as a first thing - examples are always in relation to something else and it requires a lot of thinking on how to work with independent data types. 

3 methods are great to explain.